docs/install/manage-add-ons-packages.md

```diff
@@ -0,0 +1,511 @@
+
+```{seealso}
+For a complete list of features, usage, and options, read [`cookiecutter-zope-instance`'s `README.rst`](https://github.com/plone/cookiecutter-zope-instance#readme).
+```
+
+As an example of how to configure Zope, we will add the add-on `collective.easyform` to our Zope instance.
+
+Zope is configured through the file {file}`instance.yaml` in your project.
+Modify the file as indicated.
+
+```{code-block} yaml
+:emphasize-lines: 6-9
+
+default_context:
+    initial_user_name: 'admin'
+    initial_user_password: 'admin'
+
+    load_zcml:
+        package_includes: [
+            'project_title',
+            'collective.easyform',
+        ]
+
+    db_storage: direct
+```
+
+Add-ons are listed here under the `package_includes` key to be loaded by `Zope`.
+As Python packages, they also need to be installed with `pip`.
+`pip` can install Python packages by specifying them in a requirements file.
+In your Plone project directory, open and edit the file {file}`backend/requirements.txt` as indicated.
+
+```{code-block} text
+:emphasize-lines: 3
+
+-r requirements/unreleased.txt
+src/project_title
+collective.easyform
+```
+
+```{seealso}
+Documentation of Python [Requirements File Format](https://pip.pypa.io/en/stable/reference/requirements-file-format/).
+```
+
+```{todo}
+There should be a new make target that does not require this dance of changing the working directory.
+```
+
+Then after specifying requirements, you can install them by changing your working directory to `backend`, and using the following command.
+
+```shell
+cd backend
+bin/pip install -r requirements.txt
+```
+
+Next you can use `cookiecutter` to generate the Zope configuration files.
+
+```shell
+bin/cookiecutter -f --no-input --config-file instance.yaml https://github.com/plone/cookiecutter-zope-instance
+```
+
+Let's break down that command.
+
+```{list-table} Table of Cookiecutter arguments
+:header-rows: 1
+
+* - Option
+  - Effect
+* - `-f`, `--overwrite-if-exists`
+  - Overwrite the contents of the output directory if it already exists.
+    Data in `var` stays untouched.
+* - `--no-input`
+  - Do not prompt for parameters and only use `cookiecutter.json` file content.
+* - `--config-file instance.yaml`
+  - User configuration file.
+```
+
+As we mentioned earlier in {ref}`manage-installation-details-label`, `cookiecutter-zope-instance` creates a directory {file}`instance` that contains your Zope and Plone instance configuration.
+This directory also contains persistent storage objects. During {doc}`install-from-packages`, when we ran the command `make start-backend`, the `Makefile` target ran the following command.
+
+```shell
+runwsgi -v instance/etc/zope.ini
+```
+
+That command loaded that instance's configuration, and started the Zope instance and our Plone site.
+However, if your site is still running in the backend shell session, the changes will not show up until you restart the Plone backend.
+If needed, stop your Plone backend with {kbd}`ctrl-c`.
```

Comment:
Future branch of 
cookiecutter-plone.starter branch https://github.com/collective/cookiecutter-plone-starter/tree/feature-4 will provide `make build-backend` and in short `make build` to update build after changes (like additional add-on)

Replace with
`make build`

Reply:
Yeah, I added a `make upgrade` target to do the same thing. `make build` currently tries to create a site, and it errored out. I like the new version much better. I'll run through it tonight and try it out.

Notes for me:

https://github.com/collective/cookiecutter-plone-starter/blob/2c269e73ef869389bfd5ac7e4b9296efa6b67c5d/%7B%7B%20cookiecutter.project_slug%20%7D%7D/backend/Makefile#L76-L81

Specify branch when invoking cookiecutter.

Reply:
Fixed in e27ace149791a496a1120c3eaaa2e614ea2dfbc3